src/main/java/me/totalfreedom/totalfreedommod/command/Command_wiperegions.java

```diff
@@ -69,7 +69,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         }
         else
         {
-            msg(ChatColor.RED + "There hasn't been any regions made for world \"" + world.getName() + "\".");
+            msg(ChatColor.RED + "No regions have been found for world:"" + world.getName() + "\".");

```

Comment:
add a space after the : and the double quote still needs to be escaped. keep the \